.travis.yml

```diff
@@ -1,5 +1,34 @@
 language: node_js
 node_js:
-  - '7'
-  - '6'
-  - '4'
+  - 9
+  - 8
+  - 6
+  - 4
+env:
+  - FRESH_DEPS=false
+  - FRESH_DEPS=true
```

Comment:
Set to false, then to true? What am I missing?

Reply:
Both lines are used in the test matrix, and then further on we make sure to only test with fresh deps (i.e. without `package-lock.json`) in Node.js 8.

Admittedly it's all a bit overkill for this module but it's nice to be consistent with AVA itself.

Reply:
I was confused by this too, but it creates two individual build runs.

Reply:
Ah. I get it. 👍